### DISCUSSION TITLE
Link drag and drop with cursor

### DISCUSSION THREAD

**Discussion by imericxu** [adobe/react-spectrum#6010] [General]
In desktop browsers, it’s typically expected to be able to drag and drop links, however, this feature seems to be disabled by React Aria, whether by using the `Link` component, or the `useLink` and `usePress` hooks. Any reason for this? And is there any way to reenable the default behavior?

**Comment by snowystinger** [comment]
Probably would consider this a bug. It's probably from a preventDefault. I know `usePress` has a listener for onDragStart, but it's not calling preventDefault, so will need to hunt around a bit more for why this is happening.
